.github/workflows/check-python.yaml

```diff
@@ -48,3 +48,6 @@ jobs:
 
       - name: Check docs are up to date
         run: poetry run poe docs && [ $(git status --porcelain docs/ | wc -l) -eq "0" ]
+
+      - name: Check PyInstaller compilation
+        run: poetry run poe build
```

Comment:
We probably want this in the build one rather than the check one. And then probably should add it as an artifact?

Reply:
Just wanted to have it here as an initial smoke test but yeah I agree, before we merge we should make an action for automatically building it and including zips in the release.